src/commands/diagnostic/storage.ts

```diff
@@ -33,7 +33,7 @@ export default class extends ProfileCommand {
     execute(options: ProfileOptions) {
         this.spinner.start();
 
-        const profile = this.getProfile(options);
+        const profile = this.getCurProfile();
```

Comment:
Why can't we rewrite the behavior of the ``getProfile(options)`` function? I think it could make sense to treat the logic change inside this function if possible to have fewer code changes.

Reply:
@NemTechCN

Reply:
@dgarcia360 
I've already rewrite the function `getProfile()` to replace `getCurProfile()`